1e/README.md

```diff
@@ -0,0 +1,52 @@
+# 1E
+
+## Overview
+
+The 1E Datadog Integration is an API-based integration that collects metrics from 1E products and forwards them to your Datadog account. You can use these metrics to take advantage of the out-of-the-box dashboard or you can create your own visualizations.
+
+## Setup
+
+### Configuration
+
+Getting set up with the 1E Datadog integration is as simple as emailing a Datadog API key and site to the 1E Support team. There's no further configuration required.
+
+1. Go to your [Datadog Integrations page][1] and click on the 1E tile.
+
+2. Click the **Configuration** tab and click **Install Integration** at the bottom.
+
+3. Go to your [Datadog API Keys Management page][2] and create an API key.
+
+4. Determine your [Datadog site][6] by looking at your browser's address bar:
+
+   - If the domain name is `app.datadoghq.com`, your site is `US`.
+   - If the domain name is `app.datadoghq.eu`, your site is `EU`.
```

Comment:
We have more sites than just US and EU - I'd recommend removing these two sub-bullets and prompting the user to use the documentation linked to determine their site.